src/message_interface/message_handler.rs

```diff
@@ -81,6 +81,8 @@ impl ClientMessageHandler {
 
     async fn call_client_method(&self, method: &ClientMethod) -> Result<ResponseType> {
         match method {
+            // TODO: Calling finish() here restricts binding clients to one response type, unlike the
```

Comment:
This can be removed now, we allow to get public and internal addresses as bech32 String and the conversion can be done later to hex back, no extra method needed